examples/test-app/steps/Exit/template.html

```diff
@@ -0,0 +1,46 @@
+{% extends "look-and-feel/layouts/two_thirds.html" %}
```

Comment:
I realise you didn't develop this, however, this page needs formatting - it's somewhat messy.

Reply:
perhaps raise as an issue?

Reply:
I'm raising the issue now :-)  

Surely your editor can align the HTML elements so they are at least aligned.

Reply:
thaught you were talking more about how it was written not the space indents

Reply:
:-)